src/edu/psu/sweng500/team8/coreDataStructures/Cell.java

```diff
@@ -104,4 +104,41 @@ public void load(BinaryInputStream stream) throws IOException {
 			this.type = ValueType.Given;
 		}
 	}
+
+	@Override
+	public boolean equals(Object o) {
+
+		if (null == o) {
+			return false;
+		}
+
+		if (o instanceof Cell) {
+
+			if (this.coordinates.getRowIndex() != ((Cell) o).coordinates

```

Comment:
You can shorten this a good bit.

Cell otherCell = (Cell) o;
return (this.coordinates.getRowIndex() == otherCell.getRowIndex() && this.coordinates.getColumnIndex() == otherCell.getColumnIndex() //etc ;